eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Epoch.hs

```diff
@@ -372,6 +372,6 @@ instance
 
 updateCommitteeState :: StrictMaybe (Committee era) -> CommitteeState era -> CommitteeState era
 updateCommitteeState committee (CommitteeState creds) =
-  CommitteeState $ Map.difference creds members
+  CommitteeState $ Map.restrictKeys creds (Map.keysSet members)
```

Comment:
`Map.difference` doesn't do the right thing here, but can't think of a better way to do this : only keep in `creds` the  keys that are in `members` .. Is there a better way?

Reply:
You are looking for `Map.intersection`

Reply:
Yes, great! didn't realize this worked for different value types of Map. Thanks!